client/app/account/page.tsx

```diff
@@ -18,6 +18,7 @@ export default function AccountPage() {
           buttonComponent={
             <Button
               variant={'outline'}
+              className='text-base'
```

Comment:
I think 16px is better for SP but it's okay to leave it as 18px for desktop (For all pages)

Reply:
Fixed it
31995c5bd4510e70e84bb19ecfb4b774f93ba0e4